tests/conftest.py

```diff
@@ -91,13 +91,15 @@ def remove_container():
     request.addfinalizer(remove_container)
     client = minio.get_client()
     client.make_bucket("test")
+    yield {"minio": minio}
```

Comment:
__Question(non-blocking)__: What is the advantage/reason of the `yield` here? Is it functionally required or for code clarity?

Reply:
In this case it's the equivalent, but in general using yield in fixtures is useful because it allows putting "cleanup code" after the yield.

When the function calling the fixture closes, `yield` gives control back to the fixture, whereas return does not.

Normally, yield gives a generator back, but I think (?) that pytest does some magic for fixtures to unwrap the value.

https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended